index.js

```diff
@@ -0,0 +1,6 @@
+export {
+  lspPositionToUnistPoint,
+  lspRangeToUnistPosition,
+  unistPointToLspPosition,
+  unistPositionToLspRange
```

Comment:
I recommend the names, respectively:

```
toPoint
toPosition
fromPoint
fromPosition
```

The names are rather long. Given that this package is about unist, it makes sense that my proposed names are also about unist. I think it should be clear enough

Reply:
I find it a bit awkward that the name `Position` in unist represents something other than in LSP. I agree the names are a bit long. Your suggestion is fine with me. :+1: